rust/noosphere-cli/src/native/mod.rs

```diff
@@ -178,6 +178,9 @@ pub enum ConfigGetCommand {
 
     /// Read the configured difftool command
     Difftool,
+
+    /// Get the identifier for the current sphere
+    DID,
```

Comment:
nit: prefer `Did` casing in rust outside of constants (this usually triggers a lint failure)